timeseries-streaming/timeseries-java-applications/TimeSeriesPipeline/src/main/java/com/google/dataflow/sample/timeseriesflow/verifier/TSDataPointVerifier.java

```diff
@@ -61,6 +64,14 @@ public void process(@Element TSDataPoint input, OutputReceiver<TSDataPoint> o) {
             String.format("TSDataPoint has no timestamp. %s", input.toString()));
       }
 
+      if (!(time.isAfter(GlobalWindow.TIMESTAMP_MIN_VALUE)
+          && time.isBefore(GlobalWindow.TIMESTAMP_MAX_VALUE))) {
+        throw new IllegalStateException(
+            String.format(
+                "If in Batch mode you must use outputWithTimestamp(), TSDataPoints cannot start in the global window %s",
```

Comment:
Detected TSDataPoints which are at min or max values for the Global Window. The library expects TSDataPoints with timestamps larger than GlobalWindow.TIMESTAMP_MIN_VALUE and smaller than GlobalWindow.TIMESTAMP_MAX_VALUE. The timestamp value was %s"

Reply:
Done, forgot to update the message accordingly!